### PR TITLE
Re-Brand To Bitcoin Classic

### PR DESCRIPTION
**Pull request by rusty-loy**
Cherry-picked from commits found in previous pull request #2.  

Skipped first 4 commits:
- Revert "Re-Brand ex qt & test" b6e755f  
- Revert "Update Core Copyright in Tests" ea0a127  
- Update Core Copyright in Tests ee12a2b  
- Re-Brand ex qt & test 70f6d18  

Removed modifications to src/qt/locale/*.ts files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,16 +11,16 @@ What is Bitcoin?
 Bitcoin is an experimental new digital currency that enables instant payments to
 anyone, anywhere in the world. Bitcoin uses peer-to-peer technology to operate
 with no central authority: managing transactions and issuing money are carried
-out collectively by the network. Bitcoin Core is the name of open source
+out collectively by the network. Bitcoin Classic is the name of open source
 software which enables the use of this currency.
 
 For more information, as well as an immediately useable, binary version of
-the Bitcoin Core software, see https://www.bitcoin.org/en/download.
+the Bitcoin Classic software, see https://www.bitcoin.org/en/download.
```
Comment (dskloet) [inline README.md]: There is no Bitcoin Classic software at https://www.bitcoin.org/en/download.
I don't think we should leave the documents in an inconsistent state.

File: README.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ Translations
 ------------
 
 Changes to translations as well as new translations can be submitted to
-[Bitcoin Core's Transifex page](https://www.transifex.com/projects/p/bitcoin/).
+[Bitcoin Classic's Transifex page](https://www.transifex.com/projects/p/bitcoin/).
```
Comment (dskloet) [inline README.md]: This is not Bitcoin Classic's Transifex page.

File: bitcoinclassic.creator [hunk 3]
```diff
@@ -0,0 +1 @@
+[General]
```
Comment (dskloet) [inline bitcoinclassic.creator]: What is this and how does it relate to the rebranding?

File: bitcoinclassic.config [hunk 4]
```diff
@@ -0,0 +1 @@
+// ADD PREDEFINED MACROS HERE!
```
Comment (dskloet) [inline bitcoinclassic.config]: What is this and how does it relate to the rebranding?

File: bitcoinclassic.files [hunk 5]
```diff
@@ -0,0 +1,516 @@
+src/compat/byteswap.h
```
Comment (dskloet) [inline bitcoinclassic.files]: What is this and how does it relate to the rebranding?

File: contrib/debian/manpages/bitcoin-cli.1 [hunk 6]
```diff
@@ -1,12 +1,12 @@
 .TH BITCOIN-CLI "1" "February 2015" "bitcoin-cli 0.10" 
 .SH NAME
-bitcoin-cli \- a remote procedure call client for Bitcoin Core. 
+bitcoin-cli \- a remote procedure call client for Bitcoin Classic.
```
Comment (dskloet) [inline contrib/debian/manpages/bitcoin-cli.1:3]: I suppose the client works for Core as well? Should it be worded it more generally?

File: doc/bips.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-BIPs that are implemented by Bitcoin Core (up-to-date up to **v0.10.0**):
+BIPs that are implemented by Bitcoin Classic (up-to-date up to **v0.10.0**):
```
Comment (dskloet) [inline doc/bips.md:1]: What does this mean, given that there is no Bitcoin Classic v0.10.0?

File: doc/dnsseed-policy.md [hunk 8]
```diff
@@ -43,7 +43,7 @@ related to the DNS seed operation.
 
 If these expectations cannot be satisfied the operator should
 discontinue providing services and contact the active Bitcoin
-Core development team as well as posting on
+Classic development team as well as posting on
```
Comment (dskloet) [inline doc/dnsseed-policy.md:46]: Should they contact the Classic team or the Core team, or both?
Should they post on the bitcoin-dev mailing list or somewhere else?

File: doc/release-notes.md [hunk 9]
```diff
@@ -1,4 +1,4 @@
-Bitcoin Core version 0.11.2 is now available from:
+Bitcoin Classic version 0.11.2 is now available from:
```
Comment (dskloet) [inline doc/release-notes.md:1]: No, it's not available from https://bitcoin.org/bin/bitcoin-core-0.11.2/

File: doc/release-notes.md [hunk 10]
```diff
@@ -26,7 +26,7 @@ Downgrade warning
 
 Because release 0.10.0 and later makes use of headers-first synchronization and
 parallel block download (see further), the block files and databases are not
-backwards-compatible with pre-0.10 versions of Bitcoin Core or other software:
+backwards-compatible with pre-0.10 versions of Bitcoin Classic or other software:
```
Comment (dskloet) [inline doc/release-notes.md:29]: Those version never existed as Bitcoin Classic.

File: doc/release-notes.md [hunk 11]
```diff
@@ -79,13 +79,13 @@ found at the URLs below:
 - Block versions over the last 2,000 blocks showing the days to the
   earliest possible BIP65 consensus-enforced block: <http://bitcoin.sipa.be/ver-2k.png>
 
-**Notice to miners:** Bitcoin Core’s block templates are now for
+**Notice to miners:** Bitcoin Classic’s block templates are now for
```
Comment (dskloet) [inline doc/release-notes.md:82]: This refers to prior releases of Core. It doesn't make sense to rebrand this.

File: doc/translation_process.md [hunk 12]
```diff
@@ -1,7 +1,7 @@
 Translations
 ============
 
-The Bitcoin-Core project has been designed to support multiple localisations. This makes adding new phrases, and completely new languages easily achievable. For managing all application translations, Bitcoin-Core makes use of the Transifex online translation management tool.
+The Bitcoin Classic project has been designed to support multiple localisations. This makes adding new phrases, and completely new languages easily achievable. For managing all application translations, Bitcoin Classic makes use of the Transifex online translation management tool.
```
Comment (dskloet) [inline doc/translation_process.md:4]: This isn't really true for Classic.

File: doc/translation_process.md [hunk 13]
```diff
@@ -106,6 +106,6 @@ To create a new language template, you will need to edit the languages manifest
 **Note:** that the language translation file **must end in `.qm`** (the compiled extension), and not `.ts`.
 
 ### Questions and general assistance
-The Bitcoin-Core translation maintainers include *tcatm, seone, Diapolo, wumpus and luke-jr*.You can find them, and others, in the Freenode IRC chatroom - `irc.freenode.net #bitcoin-dev`.
+The Bitcoin Classic translation maintainers include *tcatm, seone, Diapolo, wumpus and luke-jr*.You can find them, and others, in the Freenode IRC chatroom - `irc.freenode.net #bitcoin-dev`.
```
Comment (dskloet) [inline doc/translation_process.md]: None of those people have anything to do with Classic.

File: doc/translation_strings_policy.md [hunk 14]
```diff
@@ -1,7 +1,7 @@
 Translation Strings Policy
 ===========================
 
-This document provides guidelines for internationalization of the Bitcoin Core software.
+This document provides guidelines for internationalization of the Bitcoin Classic software.
```
Comment (dskloet) [inline doc/translation_strings_policy.md:4]: Is translation set up yet for Classic?

File: src/bitcoin-cli-res.rc [hunk 15]
```diff
@@ -17,7 +17,7 @@ BEGIN
         BLOCK "040904E4" // U.S. English - multilingual (hex)
         BEGIN
             VALUE "CompanyName",        "Bitcoin"
-            VALUE "FileDescription",    "bitcoin-cli (JSON-RPC client for Bitcoin Core)"
+            VALUE "FileDescription",    "bitcoin-cli (JSON-RPC client for Bitcoin Classic)"
```
Comment (dskloet) [inline src/bitcoin-cli-res.rc:20]: Actually for anything compatible with Core, right?

File: src/init.cpp [hunk 16]
```diff
@@ -899,9 +899,9 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     try {
         static boost::interprocess::file_lock lock(pathLockFile.string().c_str());
         if (!lock.try_lock())
-            return InitError(strprintf(_("Cannot obtain a lock on data directory %s. Bitcoin Core is probably already running."), strDataDir));
+            return InitError(strprintf(_("Cannot obtain a lock on data directory %s. Bitcoin Classic is probably already running."), strDataDir));
```
Comment (dskloet) [inline src/init.cpp:902]: Does Classic use a different lock from Core? Any Bitcoin client might be running, I supposed?

File: src/init.cpp [hunk 17]
```diff
@@ -1267,10 +1267,10 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
                 InitWarning(msg);
             }
             else if (nLoadWalletRet == DB_TOO_NEW)
-                strErrors << _("Error loading wallet.dat: Wallet requires newer version of Bitcoin Core") << "\n";
+                strErrors << _("Error loading wallet.dat: Wallet requires newer version of Bitcoin Classic") << "\n";
```
Comment (dskloet) [inline src/init.cpp:1270]: A newer version of Core would do as well, I guess.

File: src/net.cpp [hunk 18]
```diff
@@ -1551,7 +1551,7 @@ bool BindListenPort(const CService &addrBind, string& strError, bool fWhiteliste
     {
         int nErr = WSAGetLastError();
         if (nErr == WSAEADDRINUSE)
-            strError = strprintf(_("Unable to bind to %s on this computer. Bitcoin Core is probably already running."), addrBind.ToString());
+            strError = strprintf(_("Unable to bind to %s on this computer. Bitcoin Classic is probably already running."), addrBind.ToString());
```
Comment (dskloet) [inline src/net.cpp:1554]: Or another compatible client is running.

File: src/protocol.h [hunk 19]
```diff
@@ -68,11 +68,11 @@ class CMessageHeader
 /** nServices flags */
 enum {
     // NODE_NETWORK means that the node is capable of serving the block chain. It is currently
-    // set by all Bitcoin Core nodes, and is unset by SPV clients or other peers that just want
+    // set by all Bitcoin Classic nodes, and is unset by SPV clients or other peers that just want
```
Comment (dskloet) [inline src/protocol.h:71]: It's still also set by Core and supposedly other clients forked from Core.

File: src/qt/addresstablemodel.cpp [hunk 20]
```diff
@@ -338,7 +338,7 @@ QModelIndex AddressTableModel::index(int row, int column, const QModelIndex &par
 void AddressTableModel::updateEntry(const QString &address,
         const QString &label, bool isMine, const QString &purpose, int status)
 {
-    // Update address book model from Bitcoin core
+    // Update address book model from core
```
Comment (dskloet) [inline src/qt/addresstablemodel.cpp]: huh?

File: src/qt/bitcoin.cpp [hunk 21]
```diff
@@ -563,14 +563,14 @@ int main(int argc, char *argv[])
     /// - Do not call GetDataDir(true) before this step finishes
     if (!boost::filesystem::is_directory(GetDataDir(false)))
     {
-        QMessageBox::critical(0, QObject::tr("Bitcoin Core"),
+        QMessageBox::critical(0, QObject::tr("Bitcoin Classic"),
```
Comment (dskloet) [inline src/qt/bitcoin.cpp:566]: Maybe updating these should wait until the translations are set up correctly?

File: src/support/cleanse.h [hunk 22]
```diff
@@ -1,5 +1,6 @@
 // Copyright (c) 2009-2010 Satoshi Nakamoto
 // Copyright (c) 2009-2015 The Bitcoin Core developers
+// Copyright (c) 2014-2016 The Bitcoin Classic developers
```
Comment (dskloet) [inline src/support/cleanse.h]: 2014?

File: README.md (hunk 1 above)
Comment (rusty-loy) [inline README.md]: Should I update the download links to https://bitcoinclassic.com/ or do we already have a placeholder download page?

File: README.md (hunk 2 above)
Comment (rusty-loy) [inline README.md]: The page https://www.transifex.com/bitcoinclassic/bitcoinclassic/ exists.  Is this under control by the Classic team?

File: doc/release-notes.md (hunk 9 above)
Comment (saltduck) [inline doc/release-notes.md:1]: Suggest URL: https://bitcoinclassic.com/bin/bitcoin-core-0.11.2/

**Comment by dskloet** [comment]
I don't know. Hopefully someone more familiar with the projects can chime in.

File: doc/bips.md (hunk 7 above)
Comment (ahmedbodi) [inline doc/bips.md:1]: This is fine until we have a definitive version number for first version i think

File: src/support/cleanse.h [hunk 23]
```diff
@@ -1,5 +1,6 @@
 // Copyright (c) 2009-2010 Satoshi Nakamoto
 // Copyright (c) 2009-2015 The Bitcoin Core developers
+// Copyright (c) 2016 The Bitcoin Classic developers
```
Comment (ahmedbodi) [inline src/support/cleanse.h]: NACK. dont add any extra copyright's when a files not been changed please

**Comment by ahmedbodi** [comment]
NACK for now. Once the issues mentioned by myself have been fixed, then its an ACK by me

**Comment by ahmedbodi** [comment]
How did Jonathan's version bits commit get in here?

**Comment by jtoomim** [comment]
I think Gavin took some of my code from #48, added some of his own, fixed a few other things, and committed it.

**Comment by ahmedbodi** [comment]
Ideally it shouldnt be in this commit since this is just for the re-brand

**Comment by jtoomim** [comment]
It was merged into the bitcoinclassic:0.11.2 branch. Perhaps @rusty-loy did a git pull instead of a git rebase?

**Comment by rusty-loy** [comment]
Updated to do a rebase instead of a pull.

**Comment by rromanchuk** [comment]
I have been running this merged with 0.11.2 on mac os, no issues to report. @rusty-loy it would be great if in the near future, we could purge all mention of "QT", besides those legally required of course. It's completely nonsensical to end users.  BC currently identifies itself in the sys tray as "bitcoin-qt", it should be "Bitcoin Classic".

File: doc/bips.md (hunk 7 above)
Comment (dskloet) [inline doc/bips.md:1]: Then just leave it as Bitcoin Core for now, I would say.

File: README.md [hunk 24]
```diff
@@ -11,16 +11,16 @@ What is Bitcoin?
 Bitcoin is an experimental new digital currency that enables instant payments to
 anyone, anywhere in the world. Bitcoin uses peer-to-peer technology to operate
 with no central authority: managing transactions and issuing money are carried
-out collectively by the network. Bitcoin Core is the name of open source
+out collectively by the network. Bitcoin Classic is the name of open source
 software which enables the use of this currency.
 
 For more information, as well as an immediately useable, binary version of
-the Bitcoin Core software, see https://www.bitcoin.org/en/download.
+the Bitcoin Classic software, see https://bitcoinclassic.com/.
```
Comment (pdpi) [inline README.md:18]: We'll want this to point at the download page at some point when there is one.

File: doc/release-notes.md [hunk 25]
```diff
@@ -1,4 +1,4 @@
-Bitcoin Core version 0.11.2 is now available from:
+Bitcoin Classic version 0.11.2 is now available from:
 
   <https://bitcoin.org/bin/bitcoin-core-0.11.2/>
```
Comment (pdpi) [inline doc/release-notes.md:3]: URL needs to be fixed once we have a release.

File: doc/translation_strings_policy.md (hunk 14 above)
Comment (pdpi) [inline doc/translation_strings_policy.md:4]: Yes.

**Comment by christophebiocca** [comment]
Tested, runs fine (ArchLinux).

The main copyright notice showing up on the splash screen and the about page still says "Copyright (C) 2009-2015 The Bitcoin Core Developers". Which is technically accurate but also somewhat jarring, given that it's right below "Bitcoin Classic".

File: src/rpcnet.cpp [hunk 26]
```diff
@@ -97,7 +97,7 @@ Value getpeerinfo(const Array& params, bool fHelp)
             "    \"pingtime\": n,             (numeric) ping time\n"
             "    \"pingwait\": n,             (numeric) ping wait\n"
             "    \"version\": v,              (numeric) The peer version, such as 7001\n"
-            "    \"subver\": \"/Satoshi:0.8.5/\",  (string) The string version\n"
+            "    \"subver\": \"/Classic:0.8.5/\",  (string) The string version\n"
```
Comment (pdpi) [inline src/rpcnet.cpp:100]: How will the RPC client react to this? will it break compatibility with core client?

Comment (christophebiocca) [inline src/rpcnet.cpp:100]: This is just a help string meant for humans.

**Comment by rusty-loy** [comment]
After discussing the translation code changes with @pdpi we have decided it was safest to revert the QT_TRANSLATE_NOOP context back to bitcoin-core for now.

**Pull request merged**